README.adoc

```diff
@@ -1,39 +1,46 @@
-# Source To Image (S2I) image for Red Hat OpenShift providing OpenJDK 1.8
+# Source To Image (S2I) image for Red Hat OpenShift providing OpenJDK
 
-This is the OpenShift Source to Image (S2I) image for OpenJDK 1.8, based
-on CentOS.
+This is the OpenShift Source to Image (S2I) image for OpenJDK.
 
-## Building
+## Community images
+
+### CentOS JDK 8
 
 ```
 cekit build
 ```
 
-This will produce an image named `centos/openjdk-18-centos7`
+This will produce an image named `centos/openjdk-8-centos7`
+
+### CentOS JDK 11
 
-### JDK 11
+NOTE: This image will be available to build after CentOS 7.6 release.
 
 ```
 cekit --overrides-file=centos7-jdk11-overrides.yaml build
 ```
 
 This will produce an image named `centos/openjdk-11-centos7`
 
-### RHEL-Based Image
+## Product images
+
+NOTE: Product images for JDK > 8 are renamed.
+
+### RHEL JDK 8
 
-#### JDK 8
 ```
-cekit --overrides-file=product-overrides.yaml build
+cekit --overrides-file=rhel7-jdk8-overrides.yaml build
 ```
 
 This will produce an image named `redhat-openjdk-8/openjdk18-openshift`
 
-#### JDK 11
+### RHEL JDK 11
+
 ```
-cekit --overrides-file=product-overrides.yaml --overrides-file=rhel7-jdk11-overrides.yaml build
+cekit --overrides-file=rhel7-jdk8-overrides.yaml --overrides-file=rhel7-jdk11-overrides.yaml build
```

Comment:
should we add `--redhat` flag to this?

Reply:
I don't think so. This enables features specific to Red Hat environment, but should not really influence the image itself.